docs/src/main/tut/WorkingWithCsvParquetJson.md

```diff
@@ -68,22 +68,40 @@ val data = TypedDataset.createUnsafe[IrisLight](projectedDf)
 data.take(2).run()
 ```
 
+```scala mdoc:invisible
+spark.stop()
+```
+
 ## Working with Parquet
-```tut:invisible
-val testDataPathParquet: String = getClass.getResource("/iris.parquet").getPath
+```scala mdoc:invisible:reset-object
+import org.apache.spark.{SparkConf, SparkContext}
+import org.apache.spark.sql.SparkSession
+
+val conf = new SparkConf().setMaster("local[*]").setAppName("Frameless repl").set("spark.ui.enabled", "false")
+implicit val spark = SparkSession.builder().config(conf).appName("REPL").getOrCreate()
+spark.sparkContext.setLogLevel("WARN")
+
+import spark.implicits._
+
+val testDataPathParquet: String = "docs/src/main/resources/iris.parquet"
```

Comment:
This was the only hack I resorted too (used to be `getClass.getResource("/iris.parquet").getPath`). I couldn't figure out how mdoc interacts with classpath/resource folders.